qtmodern/windows.py

```diff
@@ -197,3 +197,180 @@ def on_titleBar_doubleClicked(self):
             self.on_btnMaximize_clicked()
         else:
             self.on_btnRestore_clicked()
+
+
+class ModernDialog(QDialog):
```

Comment:
This shares a lot of dupplicated code with `ModernWindow`, I think it would be a good idea to break the window bar out into a separate class and reuse that between the two classes.

Reply:
yeah will do this in the next few days

Reply:
Awesome! Will wait with merging it until then :)